### PR TITLE
Minor refine and fix

### PR DESCRIPTION
**Pull request by jovany-wang**
This PR contains some small refine and fixes:

(1) renaming `isEmpty` to `isNullOrEmpty`
(2) renaming `RejectedPolicys ` to `RejectedPolicies `
(3) refining some static final variables

### CODE REVIEW

**Comment by jovany-wang** [comment]
Thanks @slievrly , The CLA signed and all conflicts were resolved.

File: common/src/test/java/com/alibaba/fescar/common/util/StringUtilsTest.java [hunk 1]
```diff
@@ -41,10 +41,10 @@
      */
     @Test
     public void testIsEmpty() {
```
Comment (github-ygy) [inline common/src/test/java/com/alibaba/fescar/common/util/StringUtilsTest.java]: `public void testIsEmpty()`
better to change the method name to testIsNullOrEmpty

File: spring/src/test/java/com/alibaba/fescar/spring/annotation/BusinessProxy.java [hunk 2]
```diff
@@ -16,26 +16,33 @@
...
         System.out.println("before");
         Object result = null;
         try {
             result = method.invoke(this.proxy, args);
         } catch (Exception e) {
-
+            LOGGER.warn("Failed to invoke method {}.", method.getName());
         }
-        System.out.println("after");
+        LOGGER.debug("After invoking proxy method.");
```
Comment (slievrly) [inline spring/src/test/java/com/alibaba/fescar/spring/annotation/BusinessProxy.java]: When the logger level is debug, add the logger.enableDebug judgment to reduce the splicing of unnecessary strings.

**Comment by jovany-wang** [comment]
Thanks @github-ygy and @slievrly , all comments were addressed!

File: spring/src/test/java/com/alibaba/fescar/spring/annotation/BusinessProxy.java [hunk 3]
```diff
@@ -16,26 +16,36 @@
...
     private Object proxy;
 
     public BusinessProxy(Object proxy) {
         this.proxy = proxy;
     }
 
     @Override
-    public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
+    public Object invoke(Object proxy, Method method, Object[] args) {
+        LOGGER.debug("Before invoking proxy method.");
```
Comment (slievrly) [inline spring/src/test/java/com/alibaba/fescar/spring/annotation/BusinessProxy.java]: LOGGER.isDebugEnabled()

Comment (jovany-wang) [reply spring/src/test/java/com/alibaba/fescar/spring/annotation/BusinessProxy.java]: nice catch!

**Pull request merged**